### PR TITLE
[1881] edit course send

### PR DESCRIPTION
**Pull request by markmead**
### Context

Edit course SEND value 

Requires https://github.com/DFE-Digital/manage-courses-backend/pull/710 to be deployed.

### Before Edit
before

### During Edit
during

### After Edit
after

### CODE REVIEW

**Comment by richrace** [comment]
Using branch https://github.com/DFE-Digital/manage-courses-backend/pull/710 for the Backend this has worked.

**Comment by richrace** [comment]
[[S] Edit SEND (MCFE)](https://trello.com/c/hR6fDSPr/1881-s-edit-send-mcfe)

**Comment by dankmitchell** [comment]
Looks good, just need to resolve the conflicts

File: app/views/courses/send/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<%= content_for :page_title, "Edit SEND – #{course.name_and_code}" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_course_path(course.provider_code, course.recruitment_cycle_year, course.course_code)) %>
+<% end %>
+
+<%= render 'shared/errors' %>
+
+<h1 class="govuk-heading-xl">
+  Edit <abbr class="app-text-decoration-underline-dotted" title="Special educational needs and disability">SEND</abbr>
```
Comment (fofr) [inline app/views/courses/send/edit.html.erb]:
```suggestion
Special educational needs and disability (SEND)
```

**Pull request merged**